pytorch_addons/docs/bladedisc_torch_overview.md

```diff
@@ -0,0 +1,112 @@
+## Overview
+
+![bladedisc_torch_overview](./bladedisc_torch_overview.jpg)
+
+The figure shows how a PyTorch nn.Module is compiled via BladeDISC. In short,
+BladeDISC for PyTorch is a compiler to python subset language for deep learning.
```

Comment:
... a compiler targeting a subset of Python language for ... ?

Reply:
"a compiler to python subset language for deep learning" I am also puzzled about the meaning of this sentence:)